troposphere/static/js/components/modals/instance/image/components/Tags.react.js

```diff
@@ -56,13 +104,15 @@ define(function (require) {
             <TagMultiSelect
               models={tags}
               activeModels={imageTags}
-              onModelAdded={this.props.onTagAdded}
+              onCreateNewTag={this.onCreateNewTag}
+							onModelAdded={this.props.onTagAdded}
               onModelRemoved={this.props.onTagRemoved}
               onModelCreated={this.props.onTagCreated}
               onQueryChange={this.onQueryChange}
               width={"100%"}
               placeholderText="Search by tag name..."
               />
+						{tagCreateForm}
           </div>
         </div>
       );
```

Comment:
A mix of tabs and spaces ... it's red to show unexpected whitespace that would cause a diff-formatted patch to fail when _git_ applied it. 

I can fix this. I believe it has been fixing in `master` and I've added this as a rules to [`.eslintrc`](https://github.com/iPlantCollaborativeOpenSource/troposphere/blob/master/.eslintrc#L14) (which is run by travis-ci as a PR check).